faulthandler.c

```diff
@@ -151,6 +151,29 @@ extern const char* _Py_DumpTracebackThreads(
     PyInterpreterState *interp,
     PyThreadState *current_thread);
 
+
+#define TIMEBUFF_LEN 21 // enough space to represent a "long long" time
+static char _timebuff[TIMEBUFF_LEN] = {0};
+
+/* Return a pointer to a static buffer containing a string representation of the current unix timestamp
+ */
+char*
```

Comment:
must be static

I would prefer a function which uses a local buffer (allocated on the stack) and writes it to an fd, similar to dump_decimal() but with a wider array.